base/base/ReplxxLineReader.h

```diff
@@ -4,6 +4,12 @@
 
 #include <replxx.hxx>
 
+enum FuzzyFinderType
+{
+    FUZZY_FINDER_NONE,
+    FUZZY_FINDER_FZF,
```

Comment:
`fzf` and `skim` are rather new software projects (not universally known), let's at least add links to their repositories in a comment.